components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/GoogleOAuth2Authenticator.java

```diff
@@ -17,16 +17,24 @@
  */
 package org.wso2.carbon.identity.application.authenticator.google;
 
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdTokenVerifier;
+import com.google.api.client.http.apache.v2.ApacheHttpTransport;
+import com.google.api.client.json.gson.GsonFactory;
```

Comment:
Add these as bundle imports to the pom.

Reply:
Fixed f806cee9c82a8896103acc5729765b340d3190c1